.gitignore

```diff
@@ -2,3 +2,11 @@ out
 .idea
 .gradle
 /build/
+dal/build
+
+# Files below aren't commited, but should usually be present
```

Comment:
Does anyone ever read comments in gitignores? This remark should go in to README